src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -295,3 +307,16 @@ export function addHighlightIfYellowBgExists(node, content: string): string {
 	}
 	return content;
 }
+
+export function getUriAnchorLink(node, notebookUri: URI): URI {
+	let notebookLink: URI;
```

Comment:
technically this should be `URI | undefined`. I'd suggest just removing this and returning the values directly on 317 & 321 though.